### DISCUSSION TITLE
复杂网络条件 疑惑

### DISCUSSION THREAD

**Discussion by Ran-Xing** [jeessy2/ddns-go#792] [General]
可编辑的 DRAW.IO

这个需求是怕哪天路由器挂了，在外面连不上家里的服务器，所以做个双保险，不知道能不能这样设置

DDNS-GO 能做到吗？有example吗？

![可编辑的](https://github.com/jeessy2/ddns-go/assets/73819467/9f4abb14-5bed-4321-a471-e66849ace440)

**Comment by WaterLemons2k** [comment]
当无法通过 `cip.cc` 获取 IP 时运行命令 `ip a`：

```shell
set -o pipefail; curl -s cip.cc | grep IP | awk '{print $3}' || ip a
```

**Reply by Ran-Xing** [reply]
我不认为这是一个好的建议，也许可以添加新功能，或者已经存在？
curl --interface enp5s0 cip.cc

**Comment by Ran-Xing** [comment]
切换配置 = 只有一份配置能生效好像
